pkgs/by-name/ra/razer-cli/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  xrdb,
+}:
+
+# requires openrazer-daemon to be running on the system
+# on NixOS hardware.openrazer.enable or pkgs.openrazer-daemon
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "razer-cli";
+  version = "0-unstable-2024-04-04";
+
+  src = fetchFromGitHub {
+    owner = "lolei";
+    repo = pname;
```

`repo = pname;` is an antipattern
```suggestion
    repo = "razer-cli";
```